scripts/lldbinit.py

```diff
@@ -110,27 +115,22 @@ def print_dump(context: str, expr: str) -> None:
 
     # Try to find a type + id from the input args. If not, the id will be passed
     # through directly to C++, as it can be a variable name.
-    id_type = None
+    found_id_type = False
 
     # Look for <type><id> as a single argument.
-    if m := re.fullmatch("([a-z_]+)(\\d+)", args[1]):
+    if m := re.fullmatch("([a-z_]+)([0-9A-Fa-f]+)", args[1]):
         if m[1] in id_types:
             if len(args) != 2:
                 print_usage()
                 return
             id_type = m[1]
-            print_dump(context, f"{id_types[id_type]}({m[2]})")
-
-    # Look for <type> <id> as two arguments.
-    if not id_type:
-        if args[1] in id_types:
-            if len(args) != 3:
-                print_usage()
-                return
-            id_type = args[1]
-            print_dump(context, f"{id_types[id_type]}({args[2]})")
+            make_id_fn = id_types[id_type][0]
+            base = id_types[id_type][1]
```

Comment:
```suggestion
            (make_id_fn, base) = id_types[m[1]]
```

The GitHub-generated  delta of this is a little odd (probably because it covers a line not changed here), so here's a manual delta to show what I'm trying to suggest:

```diff
-            id_type = m[1]
-            make_id_fn = id_types[id_type][0]
-            base = id_types[id_type][1]
+            (make_id_fn, base) = id_types[m[1]]
```

Reply:
Ah yeah good idea, thanks, done